servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,44 +1,81 @@
 [[removing-service-mesh]]
 = Removing {ProductName}
 
-The following process removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
+The following process allows you to remove the {ProductShortName} from an existing {product-title} instance. Log in as a cluster admin and run these commands from any host with access to the cluster.
 
 [[remove-control-plane]]
 == Remove the control plane
 
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the operator to begin uninstalling everything it installed.
```

Comment:
@brian-avery is this note correct?  I thought that the operators watch for changes to custom resources?    

Would this be better written as "removing the custom resource triggers the operator to begin uninstalling the objects created by the custom resource"?

Reply:
Correct, the operators watch for the changes to the custom resources. The servicemesh operator is watching for the servicemeshcontrolplane and manages Kiali and Jaeger CRDs which the Kiali and Jaeger operators, respectively, use to manage the resources belonging to them.

Reply:
So it would be better to say "Removing the servicemeshcontrolplane triggers the servicemesh operator to begin uninstalling the resources it created."

Reply:
I updated this to read:

Removing the `servicemeshcontrolplane` triggers the {ProductShortName} Operator to uninstall the resources it created.